SpatialGDK/Source/SpatialGDK/Private/Interop/RPCs/SpatialRPCService.cpp

```diff
@@ -531,8 +532,37 @@ FRPCErrorInfo SpatialRPCService::ApplyRPC(const FPendingRPCParams& Params)
 	return ApplyRPCInternal(TargetObject, Function, Params);
 }
 
+namespace SpatialRPCServicePrivate
+{
+	struct NetWriteFenceResolutionHandler : FStackOnly
```

Comment:
Related to me earlier comment: The effect of this seems to be that it makes the next call to `NeedsWriteFence` return false instead of looking what else is on the stack in the net driver. Why is that important? Could you add a comment explaining this?

Reply:
Will add a comment.